src/context/product-context.js

```diff
@@ -29,7 +29,6 @@ const ProductProvider = ({ children }) => {
       (async () => {
         try {
             const { data } = await axios.get("/api/products");
-            console.log("Data:", data);
             dispatch({ type: "SUCCESS", payload: data.products });
           } catch (error) {
             dispatch({ type: "ERROR" });
```

Comment:
You can also add the error to state since, you are dispatching it

Reply:
I do have an error state before.  you can check that on line 14.